winlogbeat/docs/getting-started.asciidoc

```diff
@@ -190,6 +192,37 @@ include::{libbeat-dir}/shared/opendashboards.asciidoc[tag=open-dashboards-intro]
 
 include::{libbeat-dir}/shared/opendashboards.asciidoc[tag=open-dashboards]
 
+[float]
+[[local-user-account-setup]]
+=== Using a local non-Administrator account to run Winlogbeat
+
+By default, the +{beatname_uc}+ service runs as the `Local System` account. 
+If you want to run the +{beatname_uc}+ service as a local user account 
+that is not an Administrator, then follow the below steps. The local user account 
+must be granted `Log on as a service` in the security policy 
+and be made part of the `Builtin\Event Log Readers` group to read the event log. 
+
+. Open the Services Management console with this command:
+[source,shell]
+----------------------------------------------------------------------
+PS C:\Program Files\Winlogbeat> services.msc
+----------------------------------------------------------------------
+. Right-click on service named +{beatname_lc}+ and select `Properties`
```

Comment:
This appears to break the list numbering; "Open the serv..." is at 1. and then "Right-click on ser..." is at 1. again. Asciidoc presumably has something like four space indents to make text belong to the preceding list item.